src/diffusers/schedulers/scheduling_euler_discrete.py

```diff
@@ -216,10 +216,8 @@ def __init__(
             # FP16 smallest positive subnormal works well here
             self.alphas_cumprod[-1] = 2**-24
 
-        sigmas = np.array(((1 - self.alphas_cumprod) / self.alphas_cumprod) ** 0.5)
+        sigmas = (((1 - self.alphas_cumprod) / self.alphas_cumprod) ** 0.5).flip(0)
```

Comment:
```suggestion
        sigmas = (((1 - self.alphas_cumprod) / self.alphas_cumprod) ** 0.5).flip(0).to(dtype=torch.float32)
```

Reply:
There wasn't any upcasting in the code before. Why make it suddenly?

Reply:
The cast is unnecessary, `sigmas` are already in `float32`, coming from `alphas_cumprod`->`alphas`->`betas` where `betas` are `float32` in all cases.